### PR TITLE
 feat: added endpoint for deleting a testimonial

### PR DESCRIPTION
**Pull request by aleksifuna**
​

## Description

This PR adds an API endpoint to handle the deletion of a testimonial, it requires users to be admin and authenticated before access
​

## Related Issue (Link to issue ticket)

[FEAT]: API Endpoint to delete a testimonial #131 
​

## Motivation and Context

This route enables deleting of a testimonial based on the testimonial ID
​

## Screenshots
Get for testimonial to delete
get_test​- Pytest

Deleting the testimonial
delete

- Tested the Endpoint with Swagger UI
​

## Types of changes

- [x] New feature (Added an endpoint that deletes a testimonial from the dB)

      ​

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: api/v1/routes/testimonial.py [hunk 1]
```diff
@@ -1,18 +1,48 @@
...
+    Function for deleting a testimonial based on testimonial id
+    """
+    testimonial = db.query(Testimonial).filter(Testimonial.id == testimonial_id).first()
+    if not testimonial:
+        raise HTTPException(
+            detail="Testimonial not found",
+            status_code=status.HTTP_404_NOT_FOUND
+        )
+    db.delete(testimonial)
+    db.commit()
```
Comment (Goketech) [inline api/v1/routes/testimonial.py:35]: create a file in services folder for your database interactions

**Review by Goketech — DISMISSED** [review]

File: api/v1/routes/testimonial.py (hunk 1 above)
Comment (aleksifuna) [reply api/v1/routes/testimonial.py]: done!

**Review by Goketech — DISMISSED** [review]

File: api/v1/routes/auth.py [hunk 2]
```diff
@@ -179,4 +179,3 @@ async def verify_signin_token(token_schema: TokenRequest, db: Session = Depends(
 @auth.get("/admin")
 def read_admin_data(current_admin: Annotated[User, Depends(user_service.get_current_super_admin)]):
     return {"message": "Hello, admin!"}
-
```
Comment (Goketech) [inline api/v1/routes/auth.py:182]: revert this change

**Review by Goketech — APPROVED** [review]
nice work! check requested changes

**Review by Goketech — APPROVED** [review]

**Review by joboy-dev — APPROVED** [review]
